README.md

```diff
@@ -26,4 +26,10 @@ $ solana sign-offchain-message --keypair <keypair.json> "hey there"
 
 $ solana verify-offchain-signature --signer 4fwzf5RUcptqpMTYjNhXVuYabeX9bF1patCSYTkMufFN "hey there" 3zHMng1qyLHbPTBw1YxBVwJGBWhkqRrX6JipKqmWiT9P4oMEuacqdjvWEbHaG6CbMF6rYL4e22qkP7zcY8FP4Bfr
 Signature is valid
-```
\ No newline at end of file
+```
+
+### Sui
+Sui provides an SDK [@mysten/sui](https://www.npmjs.com/package/@mysten/sui) for signing messages.
```

Comment:
where does this page say anything about signing messages? did i miss it? can we provide an anchor to it?

Reply:
unless there is something specific on that page about message signing (which i don't see?). don't even link it